### ISSUE TITLE
Unhandled Exception: MissingPluginException(No implementation found for method getStorageDirectory on channel plugins.flutter.io/path_provider)

### ISSUE THREAD

**Issue by shahnawazahmed88** [flutter/flutter#58779]
## Steps to Reproduce

1. Install app in Marshmallow
2.Getting  Unhandled Exception: MissingPluginException(No implementation found for method getStorageDirectory on channel plugins.flutter.io/path_provider)

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel dev, 1.19.0-3.0.pre, on Linux, locale en_IN)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Android Studio (version 3.6)
[✓] Android Studio (version 4.0)
[✓] Connected device (1 available)

• No issues found!

**Comment by iapicca** [comment]
Hi @shahnawazahmed88 
Can you please provide your `flutter run --verbose` 
and your `pubspec.yaml`.
Thank you

**Comment by shahnawazahmed88** [comment]
Hi, it's working fine in the above version of Android and iOS but not working in Marshmallow getting excwption

`flutter run --verbose`

[        ] Compiling dart to kernel with 0 updated files
[   +1 ms] <- recompile package:BLUIS/main.dart 0309db6b-227b-4e3f-b0c0-0c4980b1770a
[   +1 ms] <- 0309db6b-227b-4e3f-b0c0-0c4980b1770a
[ +183 ms] E/flutter ( 3672): [ERROR:flutter/lib/ui/ui_dart_state.cc(166)] Unhandled Exception: MissingPluginException(No implementation found for method getStorageDirectory on
channel plugins.flutter.io/path_provider)
[        ] E/flutter ( 3672): #0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:157:7)
[        ] E/flutter ( 3672): <asynchronous suspension>
[        ] E/flutter ( 3672): #1      MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:332:12)
[        ] E/flutter ( 3672): #2      MethodChannelPathProvider.getExternalStoragePath (package:path_provider_platform_interface/src/method_channel_path_provider.dart:57:26)
[   +2 ms] E/flutter ( 3672): #3      getExternalStorageDirectory (package:path_provider/path_provider.dart:125:39)
[        ] E/flutter ( 3672): #4      new UAAppContext (package:BLUIS/ua_app_context.dart:70:7)
[   +1 ms] E/flutter ( 3672): #5      UAAppContext.ourInstance (package:BLUIS/ua_app_context.dart:27:47)
[        ] E/flutter ( 3672): #6      UAAppContext.ourInstance (package:BLUIS/ua_app_context.dart:27:29)
[        ] E/flutter ( 3672): #7      UAAppContext.getInstance (package:BLUIS/ua_app_context.dart:30:12)
[        ] E/flutter ( 3672): #8      _UniappMainState.setlocaleFromSharedPreference (package:BLUIS/main.dart:59:18)
[        ] E/flutter ( 3672): #9      _UniappMainState.initState (package:BLUIS/main.dart:46:5)
[        ] E/flutter ( 3672): #10     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4649:58)
[        ] E/flutter ( 3672): #11     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4485:5)
[        ] E/flutter ( 3672): #12     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3455:14)
[        ] E/flutter ( 3672): #13     Element.updateChild (package:flutter/src/widgets/framework.dart:3223:18)
[        ] E/flutter ( 3672): #14     RenderObjectToWidgetElement._rebuild (package:flutter/src/widgets/binding.dart:1132:16)
[        ] E/flutter ( 3672): #15     RenderObjectToWidgetElement.mount (package:flutter/src/widgets/binding.dart:1103:5)
[        ] E/flutter ( 3672): #16     RenderObjectToWidgetAdapter.attachToRenderTree.<anonymous closure> (package:flutter/src/widgets/binding.dart:1045:17)
[        ] E/flutter ( 3672): #17     BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2612:19)
[        ] E/flutter ( 3672): #18     RenderObjectToWidgetAdapter.attachToRenderTree (package:flutter/src/widgets/binding.dart:1044:13)
[        ] E/flutter ( 3672): #19     WidgetsBinding.attachRootWidget (package:flutter/src/widgets/binding.dart:925:7)
[        ] E/flutter ( 3672): #20     WidgetsBinding.scheduleAttachRootWidget.<anonymous closure> (package:flutter/src/widgets/binding.dart:906:7)
[        ] E/flutter ( 3672): #21     _rootRun (dart:async/zone.dart:1182:47)
[        ] E/flutter ( 3672): #22     _CustomZone.run (dart:async/zone.dart:1093:19)
[        ] E/flutter ( 3672): #23     _CustomZone.runGuarded (dart:async/zone.dart:997:7)
[        ] E/flutter ( 3672): #24     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1037:23)
[        ] E/flutter ( 3672): #25     _rootRun (dart:async/zone.dart:1190:13)
[        ] E/flutter ( 3672): #26     _CustomZone.run (dart:async/zone.dart:1093:19)
[        ] E/flutter ( 3672): #27     _CustomZone.bindCallback.<anonymous closure> (dart:async/zone.dart:1021:23)
[        ] E/flutter ( 3672): #28     Timer._createTimer.<anonymous closure> (dart:async-patch/timer_patch.dart:18:15)
[        ] E/flutter ( 3672): #29     _Timer._runTimers (dart:isolate-patch/timer_impl.dart:397:19)
[        ] E/flutter ( 3672): #30     _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:428:5)
[        ] E/flutter ( 3672): #31     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)
[        ] E/flutter ( 3672): 
[  +36 ms] E/flutter ( 3672): [ERROR:flutter/lib/ui/ui_dart_state.cc(166)] Unhandled Exception: MissingPluginException(No implementation found for method checkPermissionStatus on
channel flutter.baseflow.com/permissions/methods)
[        ] E/flutter ( 3672): #0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:157:7)
[        ] E/flutter ( 3672): <asynchronous suspension>
[        ] E/flutter ( 3672): #1      MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:332:12)
[        ] E/flutter ( 3672): #2      MethodChannelPermissionHandler.checkPermissionStatus
(package:permission_handler_platform_interface/src/method_channel/method_channel_permission_handler.dart:15:41)
[        ] E/flutter ( 3672): #3      PermissionActions.status (package:permission_handler/permission_handler.dart:25:51)
[        ] E/flutter ( 3672): #4      AskForPermission.checkAndGetPermissions (package:BLUIS/ui/helpers/ask_for_permissions.dart:27:42)
[        ] E/flutter ( 3672): #5      UAAppContext.justWait (package:BLUIS/ua_app_context.dart:104:30)
[        ] E/flutter ( 3672): <asynchronous suspension>
[        ] E/flutter ( 3672): #6      UAAppContext.initUAContext (package:BLUIS/ua_app_context.dart:89:7)
[        ] E/flutter ( 3672): #7      new UAAppContext (package:BLUIS/ua_app_context.dart:66:5)
[        ] E/flutter ( 3672): #8      UAAppContext.ourInstance (package:BLUIS/ua_app_context.dart:27:47)
[        ] E/flutter ( 3672): #9      UAAppContext.ourInstance (package:BLUIS/ua_app_context.dart:27:29)
[        ] E/flutter ( 3672): #10     UAAppContext.getInstance (package:BLUIS/ua_app_context.dart:30:12)
[        ] E/flutter ( 3672): #11     _UniappMainState.setlocaleFromSharedPreference (package:BLUIS/main.dart:59:18)
[        ] E/flutter ( 3672): #12     _UniappMainState.initState (package:BLUIS/main.dart:46:5)
[        ] E/flutter ( 3672): #13     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4649:58)
[        ] E/flutter ( 3672): #14     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4485:5)
[        ] E/flutter ( 3672): #15     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3455:14)
[        ] E/flutter ( 3672): #16     Element.updateChild (package:flutter/src/widgets/framework.dart:3223:18)
[   +1 ms] E/flutter ( 3672): #17     RenderObjectToWidgetElement._rebuild (package:flutter/src/widgets/binding.dart:1132:16)
[        ] E/flutter ( 3672): #18     RenderObjectToWidgetElement.mount (package:flutter/src/widgets/binding.dart:1103:5)
[        ] E/flutter ( 3672): #19     RenderObjectToWidgetAdapter.attachToRenderTree.<anonymous closure> (package:flutter/src/widgets/binding.dart:1045:17)
[        ] E/flutter ( 3672): #20     BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2612:19)
[        ] E/flutter ( 3672): #21     RenderObjectToWidgetAdapter.attachToRenderTree (package:flutter/src/widgets/binding.dart:1044:13)
[   +3 ms] E/flutter ( 3672): #22     WidgetsBinding.attachRootWidget (package:flutter/src/widgets/binding.dart:925:7)
[        ] E/flutter ( 3672): #23     WidgetsBinding.scheduleAttachRootWidget.<anonymous closure> (package:flutter/src/widgets/binding.dart:906:7)
[        ] E/flutter ( 3672): #24     _rootRun (dart:async/zone.dart:1182:47)
[        ] E/flutter ( 3672): #25     _CustomZone.run (dart:async/zone.dart:1093:19)
[        ] E/flutter ( 3672): #26     _CustomZone.runGuarded (dart:async/zone.dart:997:7)
[        ] E/flutter ( 3672): #27     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1037:23)
[        ] E/flutter ( 3672): #28     _rootRun (dart:async/zone.dart:1190:13)
[        ] E/flutter ( 3672): #29     _CustomZone.run (dart:async/zone.dart:1093:19)
[        ] E/flutter ( 3672): #30     _CustomZone.bindCallback.<anonymous closure> (dart:async/zone.dart:1021:23)
[        ] E/flutter ( 3672): #31     Timer._createTimer.<anonymous closure> (dart:async-patch/timer_patch.dart:18:15)
[        ] E/flutter ( 3672): #32     _Timer._runTimers (dart:isolate-patch/timer_impl.dart:397:19)
[        ] E/flutter ( 3672): #33     _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:428:5)
[        ] E/flutter ( 3672): #34     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)
[        ] E/flutter ( 3672): 
[ +189 ms] Updating files
[ +362 ms] DevFS: Sync finished
[        ] Syncing files to device Android SDK built for x86... (completed in 802ms)
[        ] Synced 2.6MB.
[        ] <- accept
[   +4 ms] Connected to _flutterView/0x9fbecc10.
[   +1 ms] Flutter run key commands.
[   +1 ms] r Hot reload. 🔥🔥🔥
[   +1 ms] R Hot restart.
[        ] h Repeat this help message.
[        ] d Detach (terminate "flutter run" but leave application running).
[   +1 ms] c Clear the screen
[        ] q Quit (terminate the application on the device).
[        ] An Observatory debugger and profiler on Android SDK built for x86 is available at: http://127.0.0.1:14991/-fyKZoZwn4E=/
[ +100 ms] E/flutter ( 3672): [ERROR:flutter/lib/ui/ui_dart_state.cc(166)] Unhandled Exception: MissingPluginException(No implementation found for method getDatabasesPath on
channel com.tekartik.sqflite)
[        ] E/flutter ( 3672): #0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:157:7)
[        ] E/flutter ( 3672): <asynchronous suspension>
[        ] E/flutter ( 3672): #1      MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:332:12)
[        ] E/flutter ( 3672): #2      invokeMethod (package:sqflite/src/sqflite_impl.dart:17:13)
[        ] E/flutter ( 3672): #3      SqfliteDatabaseFactoryImpl.invokeMethod (package:sqflite/src/factory_impl.dart:31:7)
[        ] E/flutter ( 3672): #4      SqfliteDatabaseFactoryMixin.safeInvokeMethod.<anonymous closure> (package:sqflite_common/src/factory_mixin.dart:25:35)
[        ] E/flutter ( 3672): #5      wrapDatabaseException (package:sqflite/src/exception_impl.dart:7:32)
[        ] E/flutter ( 3672): #6      SqfliteDatabaseFactoryImpl.wrapDatabaseException (package:sqflite/src/factory_impl.dart:27:7)
[        ] E/flutter ( 3672): #7      SqfliteDatabaseFactoryMixin.safeInvokeMethod (package:sqflite_common/src/factory_mixin.dart:25:7)
[        ] E/flutter ( 3672): #8      SqfliteDatabaseFactoryMixin.getDatabasesPath (package:sqflite_common/src/factory_mixin.dart:143:26)
[        ] E/flutter ( 3672): #9      getDatabasesPath (package:sqflite/sqflite.dart:168:54)
[        ] E/flutter ( 3672): #10     DatabaseHelper._initializeDatabase (package:BLUIS/db/databaseHelper.dart:66:31)
[        ] E/flutter ( 3672): #11     DatabaseHelper.database (package:BLUIS/db/databaseHelper.dart:60:25)
[        ] E/flutter ( 3672): #12     DatabaseHelper.getConfigFileList (package:BLUIS/db/databaseHelper.dart:122:30)
[        ] E/flutter ( 3672): #13     DatabaseHelper.getConfig (package:BLUIS/db/databaseHelper.dart:283:11)
[        ] E/flutter ( 3672): #14     AppMetaConfigProvider.fetchFromDB (package:BLUIS/resources/app_meta_config_provider.dart:98:66)
[        ] E/flutter ( 3672): #15     AppMetaConfigProvider.initAppMDConfig (package:BLUIS/resources/app_meta_config_provider.dart:27:45)
[        ] E/flutter ( 3672): #16     AppMetaConfigBloc.fetchAppMetaConfig (package:BLUIS/blocs/app_meta_config_bloc.dart:17:48)
[        ] E/flutter ( 3672): #17     _SplashWidgetState.appMetaConfigRequest (package:BLUIS/ui/widgets/splash_widget.dart:68:23)
[        ] E/flutter ( 3672): #18     _SplashWidgetState.initState.<anonymous closure> (package:BLUIS/ui/widgets/splash_widget.dart:172:7)
[        ] E/flutter ( 3672): #19     _rootRunUnary (dart:async/zone.dart:1198:47)
[        ] E/flutter ( 3672): #20     _CustomZone.runUnary (dart:async/zone.dart:1100:19)
[        ] E/flutter ( 3672): #21     _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
[        ] E/flutter ( 3672): #22     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
[        ] E/flutter ( 3672): #23     Future._propagateToListeners (dart:async/future_impl.dart:725:32)
[        ] E/flutter ( 3672): #24     Future._completeWithValue (dart:async/future_impl.dart:529:5)
[        ] E/flutter ( 3672): #25     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:40:15)
[        ] E/flutter ( 3672): #26     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:311:13)
[        ] E/flutter ( 3672): #27     SharedPreferenceUtil.getBoolPreference (package:BLUIS/utils/shared_preference_util.dart)
[        ] E/flutter ( 3672): <asynchronous suspension>
[        ] E/flutter ( 3672): #28     _SplashWidgetState.initState (package:BLUIS/ui/widgets/splash_widget.dart:169:10)
[        ] E/flutter ( 3672): #29     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4649:58)
[        ] E/flutter ( 3672): #30     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4485:5)
[        ] E/flutter ( 3672): #31     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3455:14)
[        ] E/flutter ( 3672): #32     Element.updateChild (package:flutter/src/widgets/framework.dart:3223:18)
[        ] E/flutter ( 3672): #33     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5839:14)
[        ] E/flutter ( 3672): #34     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3455:14)
[        ] E/flutter ( 3672): #35     Element.updateChild (package:flutter/src/widgets/framework.dart:3223:18)
[        ] E/flutter ( 3672): #36     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5839:14)
[        ] E/flutter ( 3672): #37     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3455:14)
[        ] E/flutter ( 3672): #38     Element.updateChild (package:flutter/src/widgets/framework.dart:3223:18)
[        ] E/flutter ( 3672): #39     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5839:14)
[        ] E/flutter ( 3672): #40     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3455:14)
[        ] E/flutter ( 3672): #41     Element.updateChild (package:flutter/src/widgets/framework.dart:3223:18)
[        ] E/flutter ( 3672): #42     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5839:14)
[        ] E/flutter ( 3672): #43     Element.inflateWidget (package:flutter/src/widgets/framework.da
[+2685 ms] E/flutter ( 3672): [ERROR:flutter/lib/ui/ui_dart_state.cc(166)] Unhandled Exception: MissingPluginException(No implementation found for method getDatabasesPath on
channel com.tekartik.sqflite)
[        ] E/flutter ( 3672): #0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:157:7)
[        ] E/flutter ( 3672): <asynchronous suspension>
[        ] E/flutter ( 3672): #1      MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:332:12)
[        ] E/flutter ( 3672): #2      invokeMethod (package:sqflite/src/sqflite_impl.dart:17:13)
[        ] E/flutter ( 3672): #3      SqfliteDatabaseFactoryImpl.invokeMethod (package:sqflite/src/factory_impl.dart:31:7)
[        ] E/flutter ( 3672): #4      SqfliteDatabaseFactoryMixin.safeInvokeMethod.<anonymous closure> (package:sqflite_common/src/factory_mixin.dart:25:35)
[        ] E/flutter ( 3672): #5      wrapDatabaseException (package:sqflite/src/exception_impl.dart:7:32)
[        ] E/flutter ( 3672): #6      SqfliteDatabaseFactoryImpl.wrapDatabaseException (package:sqflite/src/factory_impl.dart:27:7)
[        ] E/flutter ( 3672): #7      SqfliteDatabaseFactoryMixin.safeInvokeMethod (package:sqflite_common/src/factory_mixin.dart:25:7)
[        ] E/flutter ( 3672): #8      SqfliteDatabaseFactoryMixin.getDatabasesPath (package:sqflite_common/src/factory_mixin.dart:143:26)
[        ] E/flutter ( 3672): #9      getDatabasesPath (package:sqflite/sqflite.dart:168:54)
[        ] E/flutter ( 3672): #10     DatabaseHelper._initializeDatabase (package:BLUIS/db/databaseHelper.dart:66:31)
[        ] E/flutter ( 3672): #11     DatabaseHelper.database (package:BLUIS/db/databaseHelper.dart:60:25)
[        ] E/flutter ( 3672): #12     DatabaseHelper.getConfigFileList (package:BLUIS/db/databaseHelper.dart:122:30)
[        ] E/flutter ( 3672): #13     DatabaseHelper.getConfig (package:BLUIS/db/databaseHelper.dart:283:11)
[        ] E/flutter ( 3672): #14     AppMetaConfigProvider.fetchFromDB (package:BLUIS/resources/app_meta_config_provider.dart:98:66)
[        ] E/flutter ( 3672): #15     AppMetaConfigProvider.initAppMDConfig (package:BLUIS/resources/app_meta_config_provider.dart:27:45)
[        ] E/flutter ( 3672): #16     AppMetaConfigBloc.fetchAppMetaConfig (package:BLUIS/blocs/app_meta_config_bloc.dart:17:48)
[        ] E/flutter ( 3672): #17     _SplashWidgetState.appMetaConfigRequest (package:BLUIS/ui/widgets/splash_widget.dart:68:23)
[        ] E/flutter ( 3672): #18     _SplashWidgetState.redirectFromSplashScreen (package:BLUIS/ui/widgets/splash_widget.dart:62:7)
[        ] E/flutter ( 3672): #19     _SplashWidgetState.initState.<anonymous closure> (package:BLUIS/ui/widgets/splash_widget.dart:165:7)
[        ] E/flutter ( 3672): #20     _rootRun (dart:async/zone.dart:1182:47)
[        ] E/flutter ( 3672): #21     _CustomZone.run (dart:async/zone.dart:1093:19)
[        ] E/flutter ( 3672): #22     _CustomZone.runGuarded (dart:async/zone.dart:997:7)
[        ] E/flutter ( 3672): #23     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1037:23)
[        ] E/flutter ( 3672): #24     _rootRun (dart:async/zone.dart:1190:13)
[        ] E/flutter ( 3672): #25     _CustomZone.run (dart:async/zone.dart:1093:19)
[        ] E/flutter ( 3672): #26     _CustomZone.bindCallback.<anonymous closure> (dart:async/zone.dart:1021:23)
[        ] E/flutter ( 3672): #27     Timer._createTimer.<anonymous closure> (dart:async-patch/timer_patch.dart:18:15)
[        ] E/flutter ( 3672): #28     _Timer._runTimers (dart:isolate-patch/timer_impl.dart:397:19)
[        ] E/flutter ( 3672): #29     _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:428:5)
[        ] E/flutter ( 3672): #30     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)


#pubspec.yaml

name: BLUIS
description: A new Flutter project.

version: 1.2.0+6

environment:
  sdk: ">=2.2.0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter
  flutter_localizations:
    sdk: flutter
  flutter_cupertino_localizations: ^1.0.1

  cupertino_icons: ^0.1.2
  rxdart: ^0.22.3
  http: ^0.12.0+1
  toast: ^0.1.5
  sqflite: any
  intl: ^0.16.0
  geolocator: ^5.1.4+2
  flutter_compass: ^0.3.4
  url_launcher: ^5.2.5
  permission_handler: 5.0.0+hotfix.3
  camera: ^0.5.2+1
  firebase_core: ^0.4.0+9
  firebase_analytics: ^5.0.2
  firebase_messaging: ^6.0.13
  cloud_firestore: ^0.13.5
  file_picker: ^1.6.1
  open_file: ^3.0.1
  chewie:
    git:
      url: git://github.com/masewo/chewie.git
  video_player: ^0.10.0
  path_provider: ^1.6.10
  path: ^1.6.2
  e2e: ^0.2.0
  esys_flutter_share: ^1.0.2
  shared_preferences: ^0.5.4+5
  connectivity: ^0.4.5+3
  uuid: ^2.0.4
  logger: ^0.6.0
  event_bus: ^1.1.0
  location: ^2.3.4
  dio: ^3.0.8
  recase: ^3.0.0
  expressions: ^0.1.2
  flutter_launcher_icons: ^0.7.4
  audioplayers: ^0.14.0
  cached_network_image: ^2.0.0
  flutter_exif_rotation: ^0.2.7
  pin_code_fields: ^2.4.0
  image_picker: ^0.6.3+4
  package_info: ^0.4.0+16
  firebase_remote_config: ^0.3.0+3
  launch_review: ^2.0.0
  flutter_map:
    git:
      url: git://github.com/okaxaki/flutter_map.git
      ref: fix/support-flutter-1.10


flutter_icons:
  image_path: "assets/images/bluis_logo.png"
  android: true
  ios: false

dev_dependencies:
  flutter_test:
    sdk: flutter

**Comment by iapicca** [comment]
Hi @shahnawazahmed88 
given the error in your logs
```bash
ERROR:flutter/lib/ui/ui_dart_state.cc(166)] Unhandled Exception: MissingPluginException(No implementation found for method getStorageDirectory on
channel plugins.flutter.io/path_provider
```
I see there's an open [issue](https://github.com/flutter/flutter/issues/46679) addressing the case you described.
Please follow up on that issue, I'm closing the current one as duplicate.
If you disagree, please write in the comments and I will reopen it.
Thank you
